src/components/GetStarted/Header.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useCallback } from 'react'
+import { useHistory } from 'react-router-dom'
+// @ts-ignore
+import { GU, IconArrowRight, useTheme, useViewport } from '@aragon/ui'
+import BrandButton from '../BrandButton/BrandButton'
+
+function Header(): JSX.Element {
+  const history = useHistory()
+  const theme = useTheme()
+  const { below } = useViewport()
+  const compactMode = below('medium')
+
+  const handleNavigateToConverter = useCallback(() => {
+    history.push('/ant')
+  }, [history])
+
+  return (
+    <div
+      css={`
+        padding: ${10 * GU}px ${2 * GU}px;
+        width: 100%;
+        text-align: center;
+        max-width: ${143 * GU}px;
+        margin: auto;
+      `}
+    >
+      <h3
+        css={`
+          font-weight: 500;
```

Comment:
Similar to my comment in another PR, but we can import named weights from style/font :)